decidim-templates/app/views/decidim/templates/admin/block_user_templates/edit.html.erb

```diff
@@ -1,4 +1,9 @@
 <% add_decidim_page_title(translated_attribute(@template.name)) %>
+<div class="item_show__header">
+  <h2 class="item_show__header-title">
+    <%= t("templates", scope: "decidim.admin.titles") %>
```

Comment:
Can we give it an specific title? Something like `Edit block user message template ` or something like that. We should do the same also for the other kind of template (`Edit questionnaire template`) and also for the other action in the form ("New", so I was thinking in `New block user message template` and `New questionnaire template`). 

The problem that I'm seeing is that as the title is the same and we've lost the sidebar (that allowed to know in which kind of template you were on), now it's really difficult to know in which template you're on. I could only see that through the URL. Compare these two pages: 

* http://localhost:3000/admin/templates/questionnaire_templates/new 
* http://localhost:3000/admin/templates/block_user_templates/new

Reply:
Fixed in 8b07c0ad73